### PR TITLE
Display init parameters

### LINKED ISSUE #50: Display init parameters for Scilla contracts
We don't currently display init parameters for Scilla contracts, and should.

### PR DESCRIPTION
**Pull request by rrw-zilliqa**
Fixes #50, plus a search niggle I spotted on the way over.

### CODE REVIEW

File: src/execution/address/ScillaInitParams.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        <td className="col-span-1 text-gray-500">{valueType}</td>
+        <td className="col-span-8 text-gray-500">{value}</td>
+      </tr>
+    </>
+  );
+};
+
+export const ScillaInitParams: FC<ScillaInitParamsProps> = ({ address }) => {
+  const { zilliqa } = useContext(RuntimeContext);
+  let { data, isLoading } = useSmartContractInit(zilliqa, address);
```
Comment (saeed-zil) [inline src/execution/address/ScillaInitParams.tsx:35]: `const` is preferred if possible.

**Review by saeed-zil — APPROVED** [review]
LGTM

File: src/execution/address/ScillaInitParams.tsx (hunk 1 above)
Comment (rrw-zilliqa) [reply src/execution/address/ScillaInitParams.tsx:35]: Oops - good point well made :-)

File: src/execution/address/ScillaState.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+export const ScillaState: FC<ScillaStateProps> = ({
+  address,
+  loadContractState,
+  setLoadContractState,
+}) => {
+  const { zilliqa } = useContext(RuntimeContext);
+  let [contractState, setContractState] = useState<ContractState | null>(null);
+
+  let { data, isLoading } = useSmartContractState(
```
[the remark this answers is not in the document]
Comment (rrw-zilliqa) [reply src/execution/address/ScillaState.tsx:41]: .. and again - thanks!

**Pull request merged**